.github/workflows/build.yml

```
@@ -40,7 +40,7 @@ jobs:
           python -m pip install --upgrade pip wheel
           pip install -e .
           python -c "import autogen"
-          pip install -e. pytest
+          pip install -e.[test]
```

```suggestion
          pip install -e. pytest mock
```
minimize installation overhead.